tapir/utils/models.py

```diff
@@ -299,7 +298,7 @@ def overlapping_with(self, obj) -> DurationModelMixinQuerySet:
     def active_temporal(self, effective_date=None) -> DurationModelMixinQuerySet:
         if not effective_date:
             # if no effective date was given, use today as the default
-            effective_date = date.today()
+            effective_date = timezone.now()
```

Comment:
Should this maybe be `timezone.now().date()`? The `DurationModelMixin` uses date fields, not datetime fields. 
Though I'm not sure if dates can be timezone-aware

Reply:
Looks like the `date` class doesn't have a timezone. But do you see any problem with that?

Reply:
`datetime.date.today()` returns the local time, not UTC. I haven't thought it through, but it sounds like people could have an offset of 1 or 2 hours?!

Anyway, maybe it's parsed automatically or we ensure timezone-aware dates by `timezone.now().date()` (which is using UTC and not local time)